### PR TITLE
9 add website field to user

### PR DESCRIPTION
**Pull request by alexdovzhanyn**

### CODE REVIEW

**Comment by alexdovzhanyn** [comment]
#9

File: db/schema.rb [hunk 1]
```diff
@@ -36,22 +36,24 @@
   end
 
   create_table "users", force: :cascade do |t|
-    t.string   "email",                  default: "", null: false
-    t.string   "encrypted_password",     default: "", null: false
+    t.string   "email",                  default: "",                                                                                                                   null: false
```
Comment (yarwest) [inline db/schema.rb:39]: whats up with these tabs homie xD

**Review by yarwest — APPROVED** [review]

File: db/schema.rb (hunk 1 above)
Comment (alexdovzhanyn) [reply db/schema.rb:39]: whaddya mean?

Comment (yarwest) [reply db/schema.rb:39]: look at line 39, after 'default: "";'

Comment (alexdovzhanyn) [reply db/schema.rb:39]: that was automatically generated lol

Comment (yarwest) [reply db/schema.rb:39]: nice xD

**Pull request merged**